packages/x-codemod/src/v8.0.0/charts/rename-sparkline-colors-to-color/actual.spec.tsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-restricted-imports */
+import * as React from 'react';
+import { SparkLineChart } from '@mui/x-charts';
+
+const data = [1, 2];
+
+function Chart() {
+  const fn = (mode) => (mode === 'light' ? ['black'] : ['white']);
+
+  // prettier-ignore
+  return (
+    <React.Fragment>
+      <SparkLineChart data={data} colors={['red']} />
+      <SparkLineChart data={data} colors={fn} />
+      <SparkLineChart data={data} colors={(mode) => (mode === 'light' ? ['black'] : ['white'])} />
+      <SparkLineChart data={data} colors="red" />
```

Comment:
This is valid since `colors` can be an array of strings or a function, but added it to ensure that we handle invalid well (i.e., by not applying any transformation).

Reply:
Removed this test case because TypeScript wasn't happy with invalid properties.